scripts/firebase-admin/updateGeneralCourt.ts

```diff
@@ -0,0 +1,92 @@
+import { Script } from "./types"
+import { currentGeneralCourt } from "../../functions/src/shared"
+
+export const script: Script = async ({ db, args }) => {
+  console.log(`Update General Court ${currentGeneralCourt} in progress...`)
+  try {
+    const generalCourts = await db
+      .collection(`/generalCourts/${currentGeneralCourt}/members`)
+      .get()
+    console.log(`${generalCourts.size} members fetched`)
+
+    // check if generalCourts is valid
+    if (generalCourts.empty) {
+      console.log("ERROR: No members found in the collection.")
+      return
+    }
+
+    const courtMembersMap = new Map<string, any>()
+    for (const member of generalCourts.docs) {
+      const data = member.data()
+      // store the member data in a map with the district as the key
+      courtMembersMap.set(data.content["District"], data)
+    }
+
+    // prepare update
+    const writer = db.bulkWriter()
+
+    console.log("Fetching all profiles ...")
+    const users = await db.collection("profiles").get()
+    console.log(`${users.size} profiles fetched`)
+
+    for (const user of users.docs) {
+      let data = user.data()
```

Comment:
For convenience, you could likely use pre-existing types here, e.g.:
```ts
let data = user.data() as Profile
const currentSenator = data // This will be a ProfileMember from "components/db/profile/types.ts"
```